### DISCUSSION TITLE
Cant access database

### DISCUSSION THREAD

**Discussion by PiotrJagla** [Project-Books/book-project#995] [Q&A]
Hello!
While getting frontend and backend setup, i encoutered a problem where i cant access a databse because dbpassword password is not correct for dbuser user.
![image](https://user-images.githubusercontent.com/76881722/226721573-9e095649-c943-4e5f-a69a-77cbe2962a28.png)
i run the docker container with databse and conected to postgres in powershell terminal by running commands:
docker exec -it book_project_db bash
and in the bash:
psql -U dbuser -d book_project_db
and i can see realtions table
![image](https://user-images.githubusercontent.com/76881722/226722372-f8523221-3d0a-4cd5-8b41-de88c9ee55ce.png)
but still i cant access database in intelij because of failed authentication as in the screenshot above

Am i missing something?
Thanks in advance for help!

**Comment by tt-fan** [comment]
Hi, there. Please change password using command line:
`psql postgres ALTER USER dbuser WITH PASSWORD 'dbpassword';`
